docs/docs/behavior/hold-tap.md

```diff
@@ -59,4 +59,4 @@ If this config does not work for you, try the flavor "tap-preferred" and a short
 If you want to use a tap-hold with a keycode from a different code page, you have to define another behavior with another "bindings" parameter.For example, if you want to use SHIFT and volume up, define the bindings like `bindings = <&kp>, <&cp>;`. Only single-argument behaviors are supported at the moment.
 
 #### Note
-Astute readers may notice similarities between the possible behaviors in ZMK and other firmware, such as QMK. The hold-preferred flavor works similar to the `HOLD_ON_OTHER_KEY_PRESS` setting. The 'balanced' flavor is similar to the `PERMISSIVE_HOLD` setting, and the `tap-preferred` flavor is similar to `IGNORE_MOD_TAP_INTERRUPT`.
```

Comment:
Just one more lingering change that snuck in, then we're good to go. Thanks!

Reply:
In the rich diff view, there seems to be no changes to display. Could you confirm that?

Reply:
My suspicion is maybe it's just a line ending change?

You can try something along the lines of (assuming you have the ZMK repo as `origin` remote):

```
git checkout origin/main docs/docs/behavior/hold-tap.md
```

(For reference: https://stackoverflow.com/a/1817774)

Reply:
I am not doing on my remote machine. And i have tried every other thing except for deleting the file and re-uploading.
Man, this does makes my commit history look messed up. Sorry about that.
I will keep this in mind next time.

Reply:
No worries. Sorry you're having such trouble. No reason to hold up getting the additional docs in for this. Thanks!